program-runtime/src/invoke_context.rs

```diff
@@ -236,11 +233,7 @@ impl<'a> InvokeContext<'a> {
     }
 
     pub fn find_program_in_cache(&self, pubkey: &Pubkey) -> Option<Arc<ProgramCacheEntry>> {
```

Comment:
This method is just an alias now and can be inlined I think.

Reply:
Dropped.